docs/en/observability/synthetics-create-test.asciidoc

```diff
@@ -195,61 +209,40 @@ Journey: inline
  2 passed (2511 ms)
 ----
 
-The script can then be copied into your in your `heartbeat.yml`:
+After testing locally, you can copy the script into your
+Elastic Synthetics integration or `heartbeat.yml`.
 
-[source,yml]
-----
-heartbeat.monitors:
-- type: browser
-  id: elastic-website
-  name: Elastic website
-  schedule: "@every 1m"
-  source:
-    inline:
-      script: |-
-        step("load homepage", async () => {
-            await page.goto('https://www.elastic.co');
-        });
-        step("hover over products menu", async () => {
-            await page.hover('css=[data-nav-item=products]');
-        });
-----
-
-That's it! You can either spin up Heartbeat yourself, or jump to <<synthetics-quickstart-step-three>>
-of the Quickstart to use the provided Docker project template.
+* **{agent} and {fleet}**: Add an inline journey when configuring the Elastic Synthetics integration.
+In Monitor settings, use Monitor Type "Browser" and switch the Source Type to "Inline script".
+See <<synthetics-quickstart-fleet>> for details.
+* **{heartbeat}**: Copy and paste your test steps into `heartbeat.yml`.
+{heartbeat} spawns a separate Node.js process, schedules your tests, and runs them on a chromium browser.
```

Comment:
Based on https://www.chromium.org/chromium-projects/, I think this should be capitalized.

```suggestion
{heartbeat} spawns a separate Node.js process, schedules your tests, and runs them on a Chromium browser.
```